cassandra/cassandra-docker.sh

```diff
@@ -55,17 +55,23 @@ do
     --keystore_password_file=*)
       KEYSTORE_PASSWORD_FILE="${args#*=}"
     ;;
-    --truststore_file=*)
-      TRUSTSTORE_FILE="${args#*=}"
+    --truststore_nodes_file=* | --truststore_file=*)
+      TRUSTSTORE_NODES_FILE="${args#*=}"
     ;;
-    --truststore_password=* | --trustsotre_password=*)
-      TRUSTSTORE_PASSWORD="${args#*=}"
+    --truststore_nodes_password=* | --truststore_password=* | --trustsotre_password=*)
+      TRUSTSTORE_NODES_PASSWORD="${args#*=}"
     ;;
-    --truststore_password_file=*)
-      TRUSTSTORE_PASSWORD_FILE="${args#*=}"
+    --truststore_nodes_password_file=* | --truststore_password_file=*)
+      TRUSTSTORE_NODES_PASSWORD_FILE="${args#*=}"
     ;;
-    --cassandra_pem_file=*)
-      CASSANDRA_PEM_FILE="${args#*=}"
+    --truststore_client_file=* | --truststore_file=*)
+      TRUSTSTORE_CLIENT_FILE="${args#*=}"
+    ;;
+    --truststore_client_password=* | --trustsotre_password=*)
+      TRUSTSTORE_CLIENT_PASSWORD="${args#*=}"
+    ;;
+    --truststore_client_password_file=* | --truststore_password_file=*)
+      TRUSTSTORE_CLIENT_PASSWORD_FILE="${args#*=}"
```

Comment:
I did consider removing this, but I thought it could be useful to allow the consumer to point to a truststore/keystore, for a use case we don't predict yet. This would be specially useful if more than one CA needs to be added to the trust store.